model/preference.go

```diff
@@ -13,23 +13,24 @@ import (
 )
 
 const (
-	PREFERENCE_CATEGORY_DIRECT_CHANNEL_SHOW = "direct_channel_show"
-	PREFERENCE_CATEGORY_GROUP_CHANNEL_SHOW  = "group_channel_show"
-	PREFERENCE_CATEGORY_TUTORIAL_STEPS      = "tutorial_step"
-	PREFERENCE_CATEGORY_ADVANCED_SETTINGS   = "advanced_settings"
-	PREFERENCE_CATEGORY_FLAGGED_POST        = "flagged_post"
-	PREFERENCE_CATEGORY_FAVORITE_CHANNEL    = "favorite_channel"
-	PREFERENCE_CATEGORY_SIDEBAR_SETTINGS    = "sidebar_settings"
-
-	PREFERENCE_CATEGORY_DISPLAY_SETTINGS      = "display_settings"
-	PREFERENCE_NAME_COLLAPSED_THREADS_ENABLED = "collapsed_reply_threads"
-	PREFERENCE_NAME_CHANNEL_DISPLAY_MODE      = "channel_display_mode"
-	PREFERENCE_NAME_COLLAPSE_SETTING          = "collapse_previews"
-	PREFERENCE_NAME_MESSAGE_DISPLAY           = "message_display"
-	PREFERENCE_NAME_NAME_FORMAT               = "name_format"
-	PREFERENCE_NAME_USE_MILITARY_TIME         = "use_military_time"
-
-	PREFERENCE_CATEGORY_THEME = "theme"
+	PreferenceCategoryDirectChannelShow = "direct_channel_show"
+	PreferenceCategoryGroupChannelShow  = "group_channel_show"
+	PreferenceCategoryTutorialSteps     = "tutorial_step"
+	PreferenceCategoryAdvancedSettings  = "advanced_settings"
+	PreferenceCategoryFlaggedPost       = "flagged_post"
+	PreferenceCategoryFavoriteChannel   = "favorite_channel"
+	PreferenceCategorySidebarSettings   = "sidebar_settings"
+
+	PreferenceCategoryDisplaySettings     = "display_settings"
+	PreferenceNameCollapsedThreadsEnabled = "collapsed_reply_threads"
+	PreferenceNameChannelDisplayMode      = "channel_display_mode"
+	PreferenceNameCollapseSetting         = "collapse_previews"
+	PreferenceNameMessageDisplay          = "message_display"
+	PreferenceNameNameFormat              = "name_format"
+	PreferenceNameUseMilitaryTime         = "use_military_time"
+	PreferenceRecommendedNextSteps        = "recommended_next_steps"
```

Comment:
I thought we had removed these extra ones from the original PR?

Reply:
Oh, in `cloud` is not using camel case ? That's why this is a conflict?

Reply:
Happens when I try to cherry-pick and resolve conflicts

Reply:
Ok, it was part of a 6.0 change to not use camel case anymore https://github.com/mattermost/mattermost-server/pull/17896

To be able to merge in cloud, we need to use the previous convention and avoid this conflict. You should be able to make the change in this `cherry-pick-MM-37064-cloud` branch